client/src/Components/Questions-Answers/QuestionModal.jsx

```diff
@@ -1,7 +1,21 @@
 import React from 'react';
 
 class QuestionModal extends React.Component {
-
+  constructor(props) {
+    super(props)
+    this.state = {
+      name: '',
+      email: '',
+      question: ''
+    }
```

Comment:
Can this also be passed in as a prop by whichever question you are currently looking at?